package.json

```diff
@@ -2,11 +2,10 @@
   "name": "sheriff-monorepo",
   "private": true,
   "scripts": {
+    "dev": "turbo watch build-watch start",
```

Comment:
May I ask why you couldn't just watch build, but needed a new task? Just curious. :)

Reply:
@lishaduck actually cannot remember. Looking at it now, indeed i dont see much sense for it.

Reply:
@lishaduck actually, after revisioning this, yes, there is a reason for this.
Docs and webservices also have build command, but we dont want to rebuild them in dev, we want to just use the `start` command on them.

Reply:
Yeah, I realized that when I tried to fix it. Turbo did just add a way to do it, though it's not released yet: https://github.com/vercel/turborepo/discussions/9339#discussioncomment-11172946.

Reply:
Also, to be fair, i stopped using the `watch` command because it is broken (atleast on my Windows machine). Not sure what the problem is. It's unconvenient, but nothing major, i just have to rebuild the docs manually whenever i change a package.
I'll fix this sometimes.